test/controllers/HomeControllerSpec.scala

```diff
@@ -25,88 +25,51 @@ import play.api.mvc.{ AnyContentAsEmpty, DefaultActionBuilder, DefaultMessagesAc
 import play.api.test._
 import play.api.test.Helpers._
 import uk.gov.nationalarchives.omega.editorial.controllers.{ HomeController, SessionKeys }
-import support.BaseSpec
+import support.BaseControllerSpec
 
 /** Add your spec here. You can mock out a whole application including requests, plugins etc.
   *
   * For more information, see https://www.playframework.com/documentation/latest/ScalaTestingWithScalaTest
   */
-class HomeControllerSpec extends BaseSpec {
+class HomeControllerSpec extends BaseControllerSpec {
```

Comment:
The ticket says that this should extend `UnitTest`, but this is correct here; perhaps the ticket should be updated.